klog.go

```diff
@@ -1079,6 +1084,13 @@ func StopFlushDaemon() {
 	logging.flushD.stop()
 }
 
+// StartFlushDaemon ensures that the flush daemon runs with the given delay
+// between flush calls. If it is already running, it gets restarted.
+func StartFlushDaemon(interval time.Duration) {
+	StopFlushDaemon()
+	logging.flushD.run(interval)
+}
+
```

Comment:
This gets used in Kubernetes like this:

https://github.com/kubernetes/kubernetes/pull/108725/commits/1744e45fdd32975e38fb5b845ff5cabe0e27030c